admin/app/components/solidus_admin/orders/index/component.rb

```diff
@@ -0,0 +1,115 @@
+# frozen_string_literal: true
+
+class SolidusAdmin::Orders::Index::Component < SolidusAdmin::BaseComponent
+  def initialize(
+    page:,
+    badge_component: component('ui/badge'),
+    table_component: component('ui/table'),
+    pagination_component: component('ui/table/pagination'),
+    button_component: component("ui/button")
+  )
+    @page = page
+
+    @badge_component = badge_component
+    @table_component = table_component
+    @button_component = button_component
+    @pagination_component = pagination_component
+  end
+
+  def title
+    Spree::Order.model_name.human.pluralize
+  end
+
+  def prev_page_link
+    @page.first? ? nil : solidus_admin.url_for(host: request.host, port: request.port, **request.params, page: @page.number - 1)
+  end
+
+  def next_page_link
+    @page.last? ? nil : solidus_admin.url_for(host: request.host, port: request.port, **request.params, page: @page.next_param)
+  end
```

Comment:
Not sure if we already attempted that, but do you think that could be extracted to the page or table component, or somewhere else? Or do you think it's better to wait for something like that?

Reply:
We previously tried to move this functionality into the table component, but we found it challenging to manage. As a result, passing the links directly to the component seemed simpler.
I'd suggest keeping it here for now, at least until we finish with the features introductions on these index pages.
Just to clarify, when you mentioned the 'page component', what exactly did you have in mind?